src/Makefile

```diff
@@ -15,7 +15,7 @@ ABT_LIBDIR =  $(shell if [ -d $$ABT_DIR/lib ]; then echo "lib"; else echo "lib64
 
 # These lines may need to be adjusted for your compiler environment:
 DEBUG = -g -O0 
-CFLAGS = $(DEBUG) -fPIC -I$(ABT_DIR)/include -I$(HDF5_DIR)/include -Wall
+CFLAGS = $(DEBUG) -fPIC -I$(ABT_DIR)/include -I$(HDF5_DIR)/include -Wall -DENABLE_WRITE_MERGE
```

Comment:
-DENABLE_WRITE_MERGE should be removed/commented out to not turn on the merge feature by default, until we do more testing